hearth/lib/hearth/plugin_list.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Hearth
+  # A list of plugins that can be applied to config. Plugins are callables
+  # that take one argument (config) and are called during client initialization
+  # or operation invocation to modify config.
+  class PluginList
+    include Enumerable
```

Comment:
Why did you remove this from RBS? Did prototype include it and steep fail?

Reply:
Yeah, steep fails with it.